gui/slick/js/core.js

```diff
@@ -2453,37 +2453,38 @@ var SICKRAGE = {
             function setSeasonSceneException(data) {
                 var xemImg;
 
-                for (var season in data.seasonExceptions) {
-                    if (data.seasonExceptions.hasOwnProperty(season)) {
-                        // Check if it is a season name exception, we don't handle the show name exceptions here
-                        if (season >= 1 && Object.keys(data.xemNumbering).length > 0) {
-                            // Let's handle this as a xem season numbered exception
-                            for (var indexerSeason in data.xemNumbering) {
-                                if (data.xemNumbering.hasOwnProperty(indexerSeason) &&
-                                        data.seasonExceptions[data.xemNumbering[indexerSeason]]) {
-                                    xemImg = $('<img>', {
-                                        'id': 'xem-exception-season-' + indexerSeason,
-                                        'alt': '[xem]',
-                                        'height': '16',
-                                        'width': '16',
-                                        'src': srRoot + '/images/xem.png',
-                                        'title': data.seasonExceptions[data.xemNumbering[indexerSeason]].join(', '),
-                                    }).appendTo('[data-season=' + indexerSeason + ']');
-                                }
+                $.each(data.seasonExceptions, function(season, nameExceptions) {
+                var foundInXem = false;
+                // Check if it is a season name exception, we don't handle the show name exceptions here
+                    if (season >= 1) {
```

Comment:
I don't so but it'd be a good idea to add them if possible, some shows use stuff like "Behind the scenes Show xx - TITLE" as the name for the specials so this could mean we may actually be able to add in auto downloading of some shows. No?